### PR TITLE
[MW-2038] Remove unnecessary aliasing.

### PR DESCRIPTION
**Pull request by cbeley**
Previously discussed in https://github.com/opentable/design-tokens/issues/68.

### CODE REVIEW

**Comment by cbeley** [comment]
I didn't do it in this PR, but if the plan going forward is to share values between otkit-desktop-typography and otkit-typography (which, I'm a little confused about right now and why we are doing the aliasing to begin with with all the different names...need to catch up with design), it may make more sense to create a file 'typography-aliases.yml' instead and import from that to be clearer.

**Comment by cbeley** [comment]
Also, if we're going the functional route for colors, we could do a similar thing: color-aliases.yml to keep everything organized.

**Comment by cbeley** [comment]
So, I'll be out after Friday until the 29th.  Anything we want to change with this PR in particular?  It's moves a lot of things around, so would be better to get this out sooner than later.

Things like creating multiple aliases files can occur in other PR's.

File: OTKit/otkit-colors/token.yml [hunk 1]
```diff
@@ -1,21 +1,38 @@
...
+  # =============================================
   color-gray-utility:
-    value: "{!color-gray-utility}"
-  color-white:
-    value: "{!color-white}"
+    value: "#E1E1E1"
+
+  # Lightest gray, used for backgrounds for the page and backgrounds for active/hover states
+  # =============================================
+  color-gray-active:
```
Comment (karuto) [inline OTKit/otkit-colors/token.yml]: Thanks for adding this. This is another problem with excessive aliasing that you discovered: humans are not perfect and we sometimes forget to map a variable, and the system doesn't remind us for that. 👍

**Comment by karuto** [comment]
@cbeley I think we can go ahead and merge today once I hear your feedback, but this should be a major version bump. What do you think? Would you be able to publish?

**Comment by matteofigus** [comment]
@karuto I can publish when merged, just let me know.

**Comment by cbeley** [comment]
I think we are still good to go?  Is there something new that may have happened while I was out I'm not aware of?

**Comment by karuto** [comment]
@cbeley Not much, I think everyone's on the same page. I'm merging.

**Pull request merged**